.github/workflows/main.yml

```diff
@@ -14,8 +14,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, ubuntu-latest]
-        dafny: [2.3.0, 3.0.0, 3.1.0, 3.2.0, 3.3.0, 3.4.0]
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        dafny: [
+          2.3.0, 3.0.0, 3.1.0, 3.2.0, 3.3.0, 3.4.0, 3.4.1,
+          3.4.2, 3.5.0, 3.6.0, 3.7.0, 3.7,1, 3.7.2, 3.7.3,
+          3.8.0, 3.8.1,
```

Comment:
I think this is overkill given the number of GHA jobs it creates - my fault for not being more specific, I was confused about this list affecting the actual default behaviour.

Let's just do `[2.3.0, 3.0.0, 3.7.3, 3.8.1]`, say.

`nightly` would be nice to include but we'd have to improve `verify-dafny.sh` quite a bit so let's punt on that for now.

Reply:
```suggestion
          2.3.0, 3.0.0, 3.7.3, 3.8.1
```